frontend/src/components/EditAccount.vue

```diff
@@ -127,6 +127,7 @@ extend('email', {
 });
 
 import Dialog from '@/components/Dialog.vue';
+import config from '../config';
```

Comment:
Should this be `_config` and have a `computed` function below to return `config` (like the other view component updates)?

Reply:
In this particular case it wasn't necessary, as it's only used through the script code. I ran into conflicts with the naming when I used the name 'config' and used the variable through UX template code.